components/logins/src/store.rs

```diff
@@ -19,8 +17,8 @@ pub struct PasswordStore {
 }
 
 impl PasswordStore {
-    pub fn new(path: impl AsRef<Path>, encryption_key: Option<&str>) -> Result<Self> {
-        let db = LoginDb::open(path, encryption_key)?;
+    pub fn new(path: impl AsRef<Path>, encryption_key: &str) -> Result<Self> {
```

Comment:
For all of the below api calls that make `self.db` calls, where possible I'd move the logic to change the return type from `Login` to `PasswordInfo` to db.rs to keep these functions lean.

Reply:
I agree with this! Per our offline conversation - we'll also be changing `PasswordInfo` to `LoginRecord` since it's still needed even though the struct will be identical to `Login`

Reply:
Does it make sense at this point to always drop `Login` and only use `LoginRecord`?  Markh mentioned that it was out of scope for this ticket, but it seems like changing all the DB methods to convert between the two is almost as much work.